docs/user/dashboard/dashboard.asciidoc

```diff
@@ -4,518 +4,669 @@
+. From the editor, click *Unique count of clientip*.
 
-| image:images/treemap.png[Tree map]
+.. In the *Display name* field, enter `Unique visitors` .
 
+.. Click *Close*.
++
+[role="screenshot"]
+image::images/lens_end_to_end_1_4.png[Flyout config open]
 
-a| *Heat map*
+[float]
+==== Save the panel
 
-Displays graphical representations of data where the individual values are represented by colors. Use heat maps when your data set includes
-categorical data. For example, use a heat map to see the flights of origin countries compared to destination countries using the sample flight data.
+Save and add the visualization panel to the dashboard.
 
-| image:images/heat_map.png[Heat map]
+. From the toolbar, click *Save*.
 
-a| *Goal*
+. In the *Title* field, enter `Unique visitors`. 
 
-Displays how your metric progresses toward a fixed goal. Use the goal to display an easy to read visual of the status of your goal progression.
+. From the *Tags* drop down, select any applicable tags. 
 
-| image:images/goal.png[Goal]
+. Select *Add to Dashboard after saving*.
 
+. Click *Save and return*.
 
-a| *Gauge*
+[discrete]
+[[donut-vis]]
+=== View the distribution of visitors by operating system
 
-Displays your data along a scale that changes color according to where your data falls on the expected scale. Use the gauge to show how metric
-values relate to reference threshold values, or determine how a specified field is performing versus how it is expected to perform.
+To determine the operating systems you should continue to support, and the importance of mobile traffic from iOS devices, 
+create a donut chart that displays the top operating systems that your visitors used to access your website within the last 90 days.
 
-| image:images/gauge.png[Gauge]
+. Open *Lens*, then set the <<set-time-filter,time filter>> to *Last 90 days*.
 
+. From the *Chart Type* dropdown, select *Donut*.
 
-a| *Metric*
+. From the *Available fields* list, drag and drop the data fields to the *Drop a field or click to add* fields in the editor.
 
-Displays a single numeric value for an aggregation. Use the metric visualization when you have a numeric value that is powerful enough to tell
-a story about your data.
+.. Drag and drop *clientip* to the *Size by* field.
 
-| image:images/metric.png[Metric]
+.. Drag and drop *machine.os.keyword* to the *Slice by* field.
++
+[role="screenshot"]
+image::images/lens_end_to_end_2_1_1.png[Donut chart with clientip and machine.os.keyword fields]
 
+. Change the color palette. 
 
-a| *Data table*
+.. From the editor, click *Top values of machine.os.keyword*. 
 
-Displays your raw data or aggregation results in a tabular format. Use data tables to display server configuration details, track counts, min,
-or max values for a specific field, and monitor the status of key services.
+.. From the *Color palette* dropdown, select *Compatibility*.
```

Comment:
The compatibility palette is quite special and should only be used in specific cases (if the user wants to create a uniform dashboard and is already using Visualizations created via regular Visualize). I don't think we should advertise it in a tutorial.

My suggestions: change this to use one of the other EUI color palettes instead and add an info box below explaining the compatibility palette:

> If you already have existing Visualizations created outside of Lens and want to make sure your dashboard uses a consistent coloring scheme, select the "Compatibility" palette